Formula/ivykis.rb

```diff
@@ -0,0 +1,31 @@
+class Ivykis < Formula
+  desc "Async I/O-assisting library"
+  homepage "http://libivykis.sourceforge.net"
```

Comment:
That page leads to this, which looks like the real homepage: https://sourceforge.net/projects/libivykis/